templates/project/Cargo.toml.liquid

```diff
@@ -28,15 +28,11 @@ features = [
 ]
 
 [dependencies]
-async-channel = "2"
-embedded-hal = { version = "~0.2", features = ["unproven"]}
```

Comment:
Originally, I was going to update these to match what was in the top-level Micro-RDK repository `Cargo.toml`, but then I realized that generated projects don't actually have any direct dependencies on these crates anymore, so I've just removed them.